ironfish/src/network/peers/peerManager.ts

```diff
@@ -717,6 +727,7 @@ export class PeerManager {
         if (peer.state.type === 'CONNECTED') {
           this.updateIdentifiedPeerMap(peer)
           peer.onStateChanged.off(handler)
+          peer.send({ type: InternalMessageType.peerListRequest })
```

Comment:
This should probably be in its own handler -- This will only send the request once when the peer is initially connected/identified, and not at all if the peer is already identified. I think if we disconnect and reconnect at a later time, we'd still want to request peers right away.